### ISSUE TITLE
balance dev and training data

### ISSUE THREAD

**Issue by Tilana** [Tilana/Classification#69]
_From @Tilana on August 24, 2017 13:10_

ensure that percentage of positive and negative samples are similar in the training and the test set.

_Copied from original issue: Tilana/cnn#2_

**Comment by Tilana** [comment]
also balance positive and negative examples in the training set:
https://stackoverflow.com/questions/35155655/loss-function-for-class-imbalanced-binary-classifier-in-tensor-flow
https://blog.fineighbor.com/tensorflow-dealing-with-imbalanced-data-eb0108b10701

**Comment by Tilana** [comment]
Tensorflow Guide losses: https://www.tensorflow.org/api_guides/python/contrib.losses
